src/roslyn/eng/build.sh

```diff
@@ -174,10 +173,8 @@ while [[ $# > 0 ]]; do
     --warnaserror)
       warn_as_error=true
       ;;
-    --sourcebuild)
+    --sourcebuild|-sb)
       source_build=true
```

Comment:
Is this still used?

Reply:
The plan is that we pass the --sourcebuild switch in to repos. We can't do that yet as not all repos support it. Same for the --productbuild switch. https://github.com/dotnet/source-build/issues/4337